src/bridge/stub.c

```diff
@@ -220,7 +220,7 @@ print_rules (void)
   GList *bridges = NULL;
   CockpitTransport *transport = cockpit_interact_transport_new (0, 1, "--");
 
-  router = setup_router (NULL, &bridges);
+  router = setup_router (transport, &bridges);
```

Comment:
How is that related to the above list reverse / priority change? It looks like this rather belongs into the previous commit?